src/App/context/feature/FeatureState.js

```diff
@@ -0,0 +1,34 @@
+import React, { useReducer, useCallback } from 'react';
+import FeatureContext from './featureContext';
```

Comment:
Minor, let's have a new line between library imports and local imports just to differentiate and for better readability. We shall have code style documented so we don't lose track of these minor issues for anyone new to the project.

Reply:
Yeah that's what I would usually do, I've been in quite a hurry writing this line sorry for ommiting that.